tck-download/jakarta-expression-language-tck/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021, 2024 Contributors to the Eclipse Foundation. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.jakarta.tests.tck</groupId>
+        <artifactId>tck-download</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jakarta-expression-language-tck</artifactId>
+    <packaging>pom</packaging>
+    <name>TCK: Install Jakarta Expression Language TCK</name>
+
+    <properties>
+        <tck.test.expression-language.file>jakarta-expression-language-tck-${tck.test.expression-language.version}.zip</tck.test.expression-language.file>
+        <tck.test.expression-language.url>https://download.eclipse.org/jakartaee/expression-language/6.0/${tck.test.expression-language.file}</tck.test.expression-language.url>
+        <tck.test.expression-language.version>6.0.0</tck.test.expression-language.version>
```

Comment:
Would also be nice if this version was defined as a property in the top-level POM, to keep it and the tck runner aligned.